doc-content/enterprise-only/integration/sso-user-add-proc.adoc

```diff
@@ -22,4 +22,6 @@ For projects that deploy from {CENTRAL} on OpenShift, create an RH-SSO user call
 +
 . Define the roles as realm roles in the *Realm Roles* tab under the *Roles* section.
 +
+Alternatively, you can define the roles as client roles for the `kie` client. For instructions about configuring the `kie` client, see <<sso-central-client-create-proc>>. To use client roles, you must also configure additional settings for {CENTRAL}, as described in <<sso-client-adapter-proc>>.
```

Comment:
At some point you'll want to not use this legacy `<<anchor-id>>` format and instead use the prescribed `xref:anchor-id[]` and/or `xref:anchor-id_context-id[]` format. That legacy format has issues with context IDs, which in theory need to be in every module anyway (which I see isn't the case here yet). Just to keep on your radar.

Reply:
Thanks, added to https://issues.redhat.com/browse/BXMSDOC-7163 - I see no reason to change this in just one place,I need to do it in the entire doc as a part of its improvement (and, very likely, single-sourcing).